modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -2,20 +2,63 @@
 //
 // * operators/operator-reference.adoc
 
+ifeval::["{context}" == "cluster-operators-ref"]
+:operators:
+endif::[]
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
 [id="cluster-cloud-controller-manager-operator_{context}"]
-= Cluster Cloud Controller Manager Operator
+ifdef::operators[= Cloud Controller Manager Operator]
+ifdef::cluster-caps[= Cloud controller manager capability]
 
 [discrete]
 == Purpose
 
+ifdef::cluster-caps[]
+The Cloud Controller Manager Operator provides features for the `CloudControllerManager` capability.
+
+[NOTE]
+====
+Currently, disabling the `CloudControllerManager` capability is not supported on all platforms.
+====
+
+You can determine if your cluster supports disabling the `CloudControllerManager` capability by checking values in the cluster `Infrastructure` resource.
+
+Find the platform type for your cluster by running the following command:
+[source,terminal]
+----
+$ oc get infrastructure cluster \
+  -o jsonpath='{.status.platformStatus}'
+----
+
+* If this command returns a value of `Baremetal` or `None`, you can disable the `CloudControllerManager` capability on your cluster.
+
+* If this command returns a value of `External`, run the following command to find the state of the Cloud Controller Manager Operator:
++
+[source,terminal]
+----
+$ oc get infrastructure cluster \
+  -o jsonpath='{.status.platformStatus.External.cloudControllerManager.state}'
+----
++
+If this command returns a value of `None`, you can disable the `CloudControllerManager` capability on your cluster.
+
+[IMPORTANT]
+====
+If these commands return any other values, you cannot disable the `CloudControllerManager` capability on your cluster.
```

Comment:
```suggestion
If these commands return any other values than those listed, you cannot disable the `CloudControllerManager` capability on your cluster.
```
```suggestion
If these commands return any other values, you cannot disable the `CloudControllerManager` capability on your cluster.
```

Reply:
Not sure if I am even in love with that suggestion as I take another look :), but I will let you decide... :)